controls/radrichtextbox/import-export/Html/settings.md

```diff
@@ -0,0 +1,138 @@
+__HtmlFormatProvider__ allows for import of HTML documents and respectively export of RadRichTextBox to HTML. Additionally, the import/export settings provide modification options. The current article outlines the available settings.
+
+## Export Settings
+
+__HtmlFormatProvider__ exposes __ExportSettings__, which allow you to control the export of the RadRichTextBox document.
+
+### ExportSettings Properties
+
+* __DocumentExportLevel__: A property of type DocumentExportLevel that gets or sets which ELEMENT tags should be exported.
+	* __Document__: This is the default value. Includes the HTML declaration, more specifically &lt;HTML&gt;, &lt;TITLE&gt;, &lt;HEAD&gt; and &lt;BODY&gt; tags.
+	* __Fragment__: When set only the HTML &lt;BODY&gt; tag will be export.	
+* __ExportBoldAsStrong__: A property of type __bool__ that controls whether elements with font-weight bold are exported as <strong> tag.
+* __ExportEmptyDocumentAsEmptyString__: A property of type __bool__ that gets or sets if RadRichTextBox should export empty document as empty string.
+* __ExportFontStylesAsTags__: A property of type __bool__ that gets or sets the export font styles as tag. Specifies if &lt;i&gt;, &lt;b&gt; and &lt;u&gt; tags should be used instead of setting properties as elements of a style.
+* __ExportHeadingsAsTags__:  A property of type __bool__ that gets or sets if RadRichTextBox Heading styles should be export as html heading styles (&lt;h1&gt;, &lt;h2&gt; etc.).
+* __ExportItalicAsEm__: A property of type __bool__ that controls whether elements with font weight italic are exported as &lt;em&gt; tag.
+* __ExportLocalOrStyleValueSource__: A property of type __bool__ gets or sets if RadRichTextBox should export only properties which has local or style value source.
+* __ExportStyleMetadata__: A property of type __bool__ that set if the additional metadata should be export or not when exporting CSS classes that will be used when importing.
+* __ForbiddentPropertiesForExportCollection__: A property of type __Dictionary__ that gets the collection where you can add properties which won't be exported for certen html tag.
+* __ImageExportMode__: A property of type __ImageExportMode__ that gets or sets how the image should be exported. This property is an enumeration and it allows the following values:
+	* __None__: Image are not exported.
+	* __AutomaticInline__:  The best mode to export the image is chosen automatically.
+	* __Base64EncodedSplit__: Images are inline Base64 encoded and split into parts layed out in a table.
+	* __Base64Encoded__: Images are inline Base64 encoded.
+	* __ImageExportingEvent__: Event is fired on exporting. 
+	* __UriSource__:  The UriSource property is set as src attribute of the img tag.	
+* __PropertiesToIgnore__: A property of type __Dictionary__ that gets a collection where you can add properties which will not be exported for a certain HTML tag. The full collection of properties that can be excluded is demonstrated in __Example 1__.
```

Comment:
A property of type __Dictionary__ that allows you add properties which will not be exported for certain HTML tags....